dss_datamover/nfs_cluster.py

```diff
@@ -41,7 +41,19 @@ class NFSCluster:
     manager = Manager()
 
     def __init__(self, config={}, user_id="ansible", password="password", logger=None):
-        self.config = config.get("nfs", {})
+        self.config = config.get("fs_config", {}).get("nfs", {})
+        self.nfs_port = config.get("fs_config", {}).get("nfsport", {})
```

Comment:
All this code should be part of overriding fs_config in master_application.py. Then the code doesn't need to be changed anywhere. 
You can move all of this code to a separate function in master_application.py.
Also there is a function validate_s3_prefix in master_application.py which expects the fs_config fields to validate the prefix. That code is broken.